showCrime/dailyIncid/management/commands/harvestPatrolLog.py

```diff
@@ -302,13 +302,19 @@ def updateBoxIDTbl(boxIDTbl,lastUpdate):
 		# NB: '_' used to separate key bits; make sure it isn't already there
 		ykey = yrf.name.strip().lower().replace('_','#')
 		yrInfo = CurrBoxClient.folder(folder_id=yrf.id).get(fields=['modified_at'])
-		if ykey not in boxIDTbl['root']['kids']:
+		if ykey in boxIDTbl:
+			prevYrModDT = dateutil.parser.parse(boxIDTbl[ykey]['mdate'])
+		else:
+			assert ykey not in boxIDTbl['root']['kids'], "updateBoxIDTbl: inconsistent ykey?! %s" % (ykey)
 			boxIDTbl['root']['kids'].append(ykey)	
-		if ykey not in boxIDTbl:
 			boxIDTbl[ykey] = {'id': yrf.id,'mdate': yrInfo.modified_at,'kids': []}
+			prevYrModDT = None
+			
 		yrModDT = dateutil.parser.parse(yrInfo.modified_at)
-		prevYrModDT = dateutil.parser.parse(boxIDTbl[ykey]['mdate'])
-		if not (yrModDT > prevYrModDT and yrModDT > lastUpdate):
+		
+		if prevYrModDT != None and not (yrModDT > prevYrModDT and yrModDT > lastUpdate):
```

Comment:
I'm working on a PR to integrate [pycodestyle](https://pypi.org/project/pycodestyle/) and linting. Please use `is not None` instead of `!= None` here and elsewhere.